### PR TITLE
integrationTests: fix tests

### PR DESCRIPTION
**Pull request by AdoAdoAdo**

### CODE REVIEW

File: integrationTests/p2p/peerDisconnecting/peerDisconnecting_test.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var durationBootstrapingTime = 2 * time.Second
+var durationBootstrapingTime = 5 * time.Second
```
Comment (SebastianMarian) [inline integrationTests/p2p/peerDisconnecting/peerDisconnecting_test.go:18]: integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

File: integrationTests/p2p/peerDiscovery/kadDht/peerDiscovery_test.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var durationBootstrapingTime = 2 * time.Second
+var durationBootstrapingTime = 5 * time.Second
```
Comment (SebastianMarian) [inline integrationTests/p2p/peerDiscovery/kadDht/peerDiscovery_test.go:15]: integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

File: integrationTests/p2p/pubsub/peerReceivingMessages_test.go [hunk 3]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-var durationBootstrapingTime = 2 * time.Second
+var durationBootstrapingTime = 5 * time.Second
```
Comment (SebastianMarian) [inline integrationTests/p2p/pubsub/peerReceivingMessages_test.go:16]: integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

File: integrationTests/singleShard/block/common.go [hunk 4]
```diff
@@ -6,4 +6,4 @@ import "time"
 const StepDelay = time.Second
 
 // P2pBootstrapDelay -
-const P2pBootstrapDelay = time.Second * 12
+const P2pBootstrapDelay = time.Second * 5
```
Comment (SebastianMarian) [inline integrationTests/singleShard/block/common.go:9]: integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

File: integrationTests/singleShard/transaction/interceptedBulkTx/interceptedBulkTx_test.go [hunk 5]
```diff
@@ -17,6 +17,7 @@ import (
 )
 
 var stepDelay = time.Second
+var p2PBootstrapDelay = time.Second * 5
```
Comment (SebastianMarian) [inline integrationTests/singleShard/transaction/interceptedBulkTx/interceptedBulkTx_test.go:20]: integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

File: integrationTests/multiShard/block/common.go [hunk 6]
```diff
@@ -8,6 +8,7 @@ import (
 
 // StepDelay -
 var StepDelay = time.Second * 2
```
Comment (SebastianMarian) [inline integrationTests/multiShard/block/common.go:10]: Is this used anymore?

File: integrationTests/multiShard/block/common.go [hunk 7]
```diff
@@ -8,6 +8,7 @@ import (
...
+var P2PBootstrapDelay = time.Second * 5
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply integrationTests/multiShard/block/common.go:11]: done

File: integrationTests/p2p/peerDisconnecting/peerDisconnecting_test.go (hunk 1 above)
Comment (AdoAdoAdo) [reply integrationTests/p2p/peerDisconnecting/peerDisconnecting_test.go:18]: done

File: integrationTests/p2p/peerDiscovery/kadDht/peerDiscovery_test.go (hunk 2 above)
Comment (AdoAdoAdo) [reply integrationTests/p2p/peerDiscovery/kadDht/peerDiscovery_test.go:15]: done

File: integrationTests/p2p/pubsub/peerReceivingMessages_test.go (hunk 3 above)
Comment (AdoAdoAdo) [reply integrationTests/p2p/pubsub/peerReceivingMessages_test.go:16]: done

File: integrationTests/singleShard/block/common.go (hunk 4 above)
Comment (AdoAdoAdo) [reply integrationTests/singleShard/block/common.go:9]: done

File: integrationTests/singleShard/transaction/interceptedBulkTx/interceptedBulkTx_test.go (hunk 5 above)
Comment (AdoAdoAdo) [reply integrationTests/singleShard/transaction/interceptedBulkTx/interceptedBulkTx_test.go:20]: done

File: integrationTests/multiShard/block/common.go (hunk 6 above)
Comment (AdoAdoAdo) [reply integrationTests/multiShard/block/common.go:10]: yes seems so in three tests

**Pull request merged**